python-client/giskard/client/giskard_client.py

```diff
@@ -136,4 +143,141 @@ def create_project(self, project_key: str, name: str, description: str = None):
         actual_project_id = response.get("id")
         if actual_project_key != project_key:
             print(f"Project created with a key : {actual_project_key}")
-        return GiskardProject(self._session, actual_project_key, actual_project_id, analytics=self.analytics)
+        return Project(self._session, actual_project_key, actual_project_id, analytics=self.analytics)
+
+    def load_model_meta(self, project_key: str, uuid: str) -> ModelMeta:
+        res = self._session.get(f"project/{project_key}/models/{uuid}").json()
+        return ModelMeta(
+            name=res['name'],
+            feature_names=res['featureNames'],
+            model_type=SupportedModelTypes[res['modelType']],
+            classification_labels=res['classificationLabels'],
+            classification_threshold=res['threshold']
+        )
+
+    def load_dataset_meta(self, project_key: str, uuid: str) -> DatasetMeta:
+        res = self._session.get(f"project/{project_key}/datasets/{uuid}").json()
+        return DatasetMeta(
+            name=res['name'],
+            target=res['target'],
+            feature_types=res['featureTypes'],
+            column_types=res['columnTypes'],
```

Comment:
no, currently it's not allowed. Also I don't know if renaming features should be something we should allow, you can break the model because there could be some data preparation linked to a concrete feature name. So I'd say you have to re-upload your model if your feature names change

Reply:
Can't we separate the name of the features from the business names shown in the UI?

Our last meeting about the api, we put this as a placeholder:
```python
meta = project.get_model_metadata(model_id) -> {"name": "My Model", "class_mapping": {"a": "pretty A"}}

meta["name"] = "new name"
meta["class_mapping"] = {"a": "pretty A"}
meta["class_mapping"]["a"] ="pretty B"
meta["classification_labels"]=["Default"]

meta_ds = = project.get_dataset_metadata(ds_id)
meta_ds["column_mapping"]=...

project.save_model_metadata(model_id, meta)
```